hedera-node/test-clients/src/main/java/com/hedera/services/bdd/junit/support/validators/block/BlockContentsValidator.java

```diff
@@ -65,50 +68,106 @@ public void validateBlocks(@NonNull final List<Block> blocks) {
     }
 
     private static void validate(Block block) {
-        final var blockItems = block.items();
+        final var items = block.items();
+        if (items.isEmpty()) {
+            Assertions.fail("Block is empty");
+        }
 
-        // A block SHALL start with a `header`.
-        if (!blockItems.getFirst().hasBlockHeader()) {
-            Assertions.fail("Block does not start with a block header");
+        if (items.size() <= 2) {
+            Assertions.fail("Block contains only header and proof with no rounds");
```

Comment:
In theory a (very) invalid block could contain other block items. This sentence assumes a header and block proof items without actually verifying the types of those block items. 

```suggestion
            Assertions.fail("Block contains insufficient number of block items");
```

Reply:
Updated this